text.html

```diff
@@ -74,136 +88,198 @@ <h2 class="title">Tonic</h2>
                             <li><img class="counter" src="Counter.png"></li>
                             <li>2015</li>
                         </ul>
-                        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <p class="description">A daily selection of privately personalized reads; no accounts or
+                            sign-ups required.</p>
                         <ul class="languages">
                             <li class="stack">html</li>
                             <li class="stack">css</li>
                             <li class="stack">javascript</li>
                         </ul>
-                        <button class="btn">See Project </button>  
-                    </div> 
+                        <button class="btn">See Project </button>
+                    </div>
                 </article>
                 <article class="card2">
-                    <div><img src="Snapshoot Portfolio (1).png"></div>
+                    <div><img src="Snapshoot Portfolio (3).png"></div>
                     <div class="elements2">
                         <h2 class="title">Multi-Post Stories</h2>
                         <ul class="highlights">
-                            <li>CANOPY</li>
+                            <li>FACEBOOK</li>
                             <li><img class="counter" src="Counter.png"></li>
-                            <li>Back End Dev</li>
+                            <li>Full Stack Dev</li>
                             <li><img class="counter" src="Counter.png"></li>
                             <li>2015</li>
                         </ul>
-                        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <p class="description">Experimental content creation feature that allows users to add to an
+                            existing story over the course of a day without spamming their friends.</p>
                         <ul class="languages">
                             <li class="stack">html</li>
+                            <li class="stack">Ruby on rails</li>
                             <li class="stack">css</li>
                             <li class="stack">javascript</li>
                         </ul>
```

Comment:
Kindly try to fix this behavior in the mobile version, these tags are coming over the description of the project, kindly fix this for all the projects 🤗

![image](https://user-images.githubusercontent.com/73714615/192990223-f3f13f2f-97af-46ca-a792-8fd8867220a5.png)